frontend/src/MakerClient.tsx

```diff
@@ -0,0 +1,41 @@
+export interface CfdSellOrderPayload {
+    price: number;
+    min_quantity: number;
+    max_quantity: number;
+}
+
+export interface AcceptOrderRequestPayload {
+    order_id: string;
+}
+
+export async function postCfdSellOrderRequest(payload: CfdSellOrderPayload) {
+    let res = await fetch(`/api/order/sell`, { method: "POST", body: JSON.stringify(payload), credentials: "include" });
+
+    if (!res.status.toString().startsWith("2")) {
+        console.log("Status: " + res.status + ", " + res.statusText);
+        throw new Error("failed to publish new order");
+    }
+}
+export async function postAcceptOrder(payload: AcceptOrderRequestPayload) {
+    let res = await fetch(
+        `/api/order/accept`,
+        { method: "POST", body: JSON.stringify(payload), credentials: "include" },
+    );
+
+    if (!res.status.toString().startsWith("2")) {
+        console.log("Status: " + res.status + ", " + res.statusText);
+        throw new Error("failed to publish new order");
```

Comment:
This error text seems to be off.

Reply:
Also, looking at the Rust code, this route will never return an error so I think we don't need to do any of this?

Reply:
It can fail if you lose connection to your application.

Reply:
You are checking the HTTP status code here. That implies that you have a connection. If you don't have a connection, the call to `fetch` would have already thrown an exception.